packages/squidex/schema/crn/schemas/users.json

```diff
@@ -9,7 +9,7 @@
     },
     "scripts": {
       "create": "if(JSON.stringify(ctx.data.teams.iv).indexOf('ASAP Staff') > 0  && ctx.data.role.iv !== 'Staff') {\n    reject('Only Staff can be given this team role. Change ASAP Hub Role to \"Staff\".')\n}",
-      "update": "if(JSON.stringify(ctx.data.teams.iv).indexOf('ASAP Staff') > 0  && ctx.data.role.iv !== 'Staff') {\n    reject('Only Staff can be given this team role. Change ASAP Hub Role to \"Staff\".')\n}"
+      "update": "if(JSON.stringify(ctx.data.teams.iv).indexOf('ASAP Staff') > 0  && ctx.data.role.iv !== 'Staff') {\n    reject('Only Staff can be given this team role. Change ASAP Hub Role to \"Staff\".')\n}\n\nconst becameAlumni = ctx.data.alumniSinceDate.iv && (ctx.oldData.alumniSinceDate.iv !== ctx.data.alumniSinceDate.iv);\nlet anyTeamChanged = false;\n\nctx.data.teams.iv = ctx.data.teams.iv.map((team) => {\n    if (becameAlumni && !team.inactiveSinceDate) {\n        team.inactiveSinceDate = new Date().toISOString();\n        anyTeamChanged = true;\n    }\n    return team;\n});\n\nif (anyTeamChanged) {\n    replace();\n}"
```

Comment:
there is no way to test it really so i assume this has been manually tested and it works? was it copied from somewhere?

Reply:
correct, it was tested and copied from https://cloud.squidex.io/app/crn-2528/schemas/users?tab=scripts